code/components/citizen-devtools/src/NetEventLog.cpp

```diff
@@ -92,17 +124,30 @@ static InitFunction initFunction([]()
 		{
 			if (eventSource.find("net:") == 0)
 			{
+#if IS_FXSERVER
+				EventLog_Add(eventName, eventPayload.size(), false, std::optional{ eventSource.substr(4) });
+#else
 				EventLog_Add(eventName, eventPayload.size(), false);
+#endif
+				
 			}
 		});
+
+		eventManager->OnClientEventTriggered.Connect([](const std::string_view& eventName, size_t& eventSize, const std::optional<std::string_view>& eventTarget)
+		{
+			EventLog_Add(std::string{ eventName }, eventSize, true, std::string{ eventTarget.value_or("-1") });
```

Comment:
Instead of showing the target as a number, maybe showing their name (plus a shortcut to the `citizen:server:gui` player list entry for them?) could work.

(similarly, instead of enabling this for the server in `citizen:devtools`, one could just `files` it in `citizen:server:gui` using the COMPILING_CITIZEN_SERVER_IMPL define, solving the callout issue as well)